content/pt/docs/guias/APROVACAO/Aprovando-uma-transacao/_index.md

```diff
@@ -4,7 +4,7 @@ linkTitle: "Aprovando uma Transação"
 date: 2021-06-28T09:00:00-03:00
 lastmod: 2021-06-28T09:00:00-03:00
 weight: 2
-draft: false
+draft: true
```

Comment:
those pages with empty content but marked with the draft flag will be removed later since they are being covered on APROVACAO? if so, wouldn't be better to delete it?

Reply:
agree! Deleted them